tests/spread/ci/build-lxd/task.yaml

```diff
@@ -17,13 +17,20 @@ execute: |
     gcloud auth activate-service-account --key-file="./sa.json"
 
     # launch the new lxc instance from an ubuntu 22.04 base
+    # FIXME: switch to noble when available
     lxc launch images:ubuntu/mantic $INSTANCE_NAME
```

Comment:
I jus tried `lxc launch images:ubuntu/noble`, seems to be creating an image now.

Reply:
Done!